src/components/DataCollector/index.js

```diff
@@ -12,7 +12,10 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const createDataCollector = ({ eventManager }) => {
+const CONFIG_DOC_URI =
+  "https://launch.gitbook.io/adobe-experience-platform-web-sdk/fundamentals/executing-commands";
```

Comment:
I think it's time we use the bitly style links for our docs. This link for example is outdated because our docs moved here: https://www.adobe.io/apis/experienceplatform/home/services/web-sdk.html

@Aaronius was working on setting the links up, let's triple check with him; if not, I'll create a ticket and start that process. In any case, let's change this link to the `adobe.io` one pls.